--- DVDVideoCodecIMX.cpp.old	2016-02-19 23:45:14.446968592 +0100
+++ DVDVideoCodecIMX.cpp	2016-02-19 23:46:16.152593420 +0100
@@ -679,6 +679,14 @@ void CDVDVideoCodecIMX::Dispose()
   return;
 }
 
+inline
+void CDVDVideoCodecIMX::SetSkipMode()
+{
+  VpuDecSkipMode skip = m_dropRequest ? VPU_DEC_SKIPB : VPU_DEC_SKIPNONE;
+  if (VPU_DEC_RET_SUCCESS != VPU_DecConfig(m_vpuHandle, VPU_DEC_CONF_SKIPMODE, &skip))
+    CLog::Log(LOGERROR, "%s - iMX VPU set skip mode failed.\n", __FUNCTION__);
+}
+
 int CDVDVideoCodecIMX::Decode(BYTE *pData, int iSize, double dts, double pts)
 {
   VpuDecFrameLengthInfo frameLengthInfo;
